integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextmove/message/FileMessagePersister.java

```diff
@@ -25,15 +25,23 @@ public FileMessagePersister(IntegrasjonspunktProperties props) {
         this.props = props;
     }
 
+    private boolean patchIncommingMessage = true;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "patchIncommingMessage" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1213)